content/source/docs/enterprise/private/preflight-installer.html.md

```diff
@@ -12,12 +12,14 @@ Before installing the Private Terraform Enterprise software, you'll need to prep
 
 * TLS private key and certificate
   * The installer allows for using a certificate signed by a public or private CA. If you do not use a trusted certificate, your VCS provider will likely reject that certificate when sending webhooks. The key and X.509 certificate should both be PEM (base64) encoded.
-* License key (provided by HashiCorp)
+* License file (provided by HashiCorp)
 
 ~> **Note:** If you use a certificate issued by a private Certificate
    Authority, you must provide the certificate for that CA in the
    `Certificate Authority (CA) Bundle` section of the installation. This allows services
    running within PTFE to access each other properly.
+   See [Installation: Trusting SSL/TLS Certificates](./install-installer#trusting-ssl-tls-certificates)
```

Comment:
OH WAIT, this is a broken link. Needs a `.html` in there! 

Once that's fixed, my 👍 is real. 😅

Reply:
Fixed!